znai-server/src/main/java/com/twosigma/znai/server/clean/FileCleanHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twosigma.znai.server.clean;
+
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpServerRequest;
+
+public class FileCleanHandler implements Handler<HttpServerRequest> {
```

Comment:
Is this for a single file or for the entire documentation?

Reply:
Entire document.
I just follow the convention like FileUploadHandler.

Reply:
I see. In case of `FileUpload` we upload a single zip file. But when we delete docs it removes more than just one.

Reply:
I will change FileUpload to DocumentUpload and FileClean to DocumentRemove then.

Reply:
That makes sense. DocumentationUpload may work better.